sickbeard/common.py

```diff
@@ -202,15 +202,15 @@ class Quality(object):
         HDTV: "720p HDTV",
         RAWHDTV: "1080i HDTV",
         FULLHDTV: "1080p HDTV",
-        HDWEBDL: "720p WEB-DL",
-        FULLHDWEBDL: "1080p WEB-DL",
+        HDWEBDL: "720p WEB",
+        FULLHDWEBDL: "1080p WEB",
```

Comment:
i don't agree with this. its `HDWEBDL`.  WEB is scene only.  WEB-DL is from iitunes- quality is better than scene